extension-types.md

```diff
@@ -38,18 +38,32 @@ metadata).
 ## Extension metadata
 
 When GeoArrow-encoded Arrays have the `ARROW:extension:metadata` metadata
-field set, it must be seriaized as a UTF-8 encoded JSON object. The following
-keys in the JSON metadata object are supported:
-
-- `crs`: A JSON object describing the coordinate reference system (CRS)
-  using [PROJJSON](https://proj.org/specifications/projjson.html).
-  This key can also be omitted if the producer does not have any
-  information about the CRS. Note that regardless of the axis
-  order specified by the CRS, axis order will be interpreted
+field set, it must be seriaized as a UTF-8 encoded JSON object. The extension
+metadata specification is intentionally aligned with the
+[GeoParquet column metadata specifification](https://github.com/opengeospatial/geoparquet/blob/main/format-specs/geoparquet.md#metadata).
```

Comment:
This still locks users into a single format of PROJJSON which is a vendor lock.  I would encourage you to be more flexible.

Reply:
Did you read the changes?

> A string containing an undefined CRS representation.

That is not PROJJSON.

Reply:
This draft may change again depending on whether http://github.com/apache/parquet-format/pull/240 gets merged, which itself 
prioritizes but does not mandate PROJJSON.